src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/ExplainIT.java

```diff
@@ -202,6 +202,17 @@ public void explainNLJoin() throws IOException {
         Assert.assertThat(result.replaceAll("\\s+", ""), equalTo(expectedOutput.replaceAll("\\s+", "")));
     }
 
+    @Test
+    public void columnAliasInWhere() throws IOException {
```

Comment:
Could you add another IT (maybe in `QueryIT`) to make sure the execution works too?